calico_containers/pycalico/netns.py

```diff
@@ -51,6 +51,20 @@ def setup_logging(logfile, level=logging.INFO):
     _log.addHandler(handler)
 
 
+def increment_metrics(namespace):
+    """
+    Increment the metric of any default route that has a metric of 0.
+    :param namespace: The Networking namespace of the container.
+    :return: None. Raises CalledProcessError on error.
+    """
+    with NamedNamespace(namespace) as ns:
```

Comment:
Makes sense. but what's the importance of reverse order? Is it to prevent the algorithm from breaking when, during execution, a route is bumped to the same metric of anotther route? Since the routing table is snapshotted (`ns.check_output`), it should be fine, unless the routing table doesn't allow two routes with the same metric?